oaknational/Oak-Web-Application
# [CUR-454] fix: added `aria-labelledby` to `radiogroup`s in subject-phase picker

## Description

Music year: 1990

- added `aria-labelledby` to `radiogroup`s in subject-phase picker
- remove old `aria-label` tags from `radiogroup`s

## Issue(s)

Fixes `CUR-454`

## How to test

1. Head over to `/teachers/curriculum`
2. Use the subject phase picker with voice over enabled
3. With voiceover check the following radiogroups get announced by their titles
    - "Curriculum plans"
    - "Choose a school phase"
    - "Choose an exam board for KS4"

## Checklist
